packages/theme-chalk/src/tabs.scss

```
@@ -32,6 +32,43 @@
+      & ~ .is-active::after {
+        content: '';
+        position: absolute;
+        left: 20px;
+        right: 20px;
+        height: 2px;
+        bottom: 0;
+        z-index: 1;
+        background-color: getCssVar('color-primary');
+      }
+
+      &.is-bottom ~ .is-active::after {
```

Stylelint (17.14.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)